astropy/modeling/utils.py

```diff
@@ -484,3 +484,64 @@ def combine_labels(left, right):
 
     return left + right
 
+
+def render_models2sky(model_list, image, x_key='x_mean', y_key='y_mean'):
```

Comment:
Like the bounding box, the location is something the models should declare, so that you don't have to guess parameter names.